keyboards/3.0/fr-t-k0-azerty.xml

```diff
@@ -10,6 +10,7 @@
 	<info author="Team Keyboard" normalization="NFC" layout="AZERTY" indicator="FR" />
 	<names>
 		<name value="French" />
+		<name value="French AZERTY" /> <!-- a second name just to expose an issue -->
```

Comment:
Added the second name just to test and expose this issue.

```suggestion
		<name value="French AZERTY" />
```

The comment can stay in this PR -- it won't be useful for future readers of the code.